src/evaluate/evaluator/base.py

```diff
@@ -267,6 +267,12 @@ def compute(
             random_state=random_state,
         )
 
+        # TODO: To clarify why `wer` and `cer` return float
```

Comment:
That's an oversight in the standardization of the metrics. Let me fix that in a separate PR so we can remove this here.

Reply:
Actually that will probably break a few things so let's keep your workaround for now.

Reply:
Let's actually keep your workaround for now.